### ISSUE TITLE
Special characters (accents) not managed in alpha & alpha_numeric validators

### ISSUE THREAD

**Issue by ben-s** [Wixel/GUMP#4]
Hi,
I'm using your great class for a subscription form on my website.
Everything is working well, but I'm designing french websites and I need to accept accentuated characters for alpha and alpha_numeric validators.

Here are my own versions of the default gump validators :
<code>protected static function validate_alpha($field, $input, $param = NULL)
    {
        if(!isset($input[$field]))
        {
            return;
        }

```
    if(!preg_match("/^([a-zÀÁÂÃÄÅÇÈÉÊËÌÍÎÏÒÓÔÕÖÙÚÛÜÝàáâãäåçèéêëìíîïðòóôõöùúûüýÿ])+$/i", $input[$field]) !== FALSE)
    {
        return array(
            'field' => $field,
            'value' => $input[$field],
            'rule'  => __FUNCTION__
        );
    }
}

/**
 * Determine if the provided value contains only alpha-numeric characters
 * 
 * @static
 * @access protected
 * @param  string $field
 * @param  array $input
 * @return mixed
 */ 
protected static function validate_alpha_numeric($field, $input, $param = NULL)
{   
    if(!isset($input[$field]))
    {
        return;
    }

    if(!preg_match("/^([a-z0-9ÀÁÂÃÄÅÇÈÉÊËÌÍÎÏÒÓÔÕÖÙÚÛÜÝàáâãäåçèéêëìíîïðòóôõöùúûüýÿ])+$/i", $input[$field]) !== FALSE)
    {
        return array(
            'field' => $field,
            'value' => $input[$field],
            'rule'  => __FUNCTION__
        );
    }
}

/**
 * Determine if the provided value contains only alpha characters with dashed and underscores
 * 
 * @static
 * @access protected
 * @param  string $field
 * @param  array $input
 * @return mixed
 */
protected static function validate_alpha_dash($field, $input, $param = NULL)
{
    if(!isset($input[$field]))
    {
        return;
    }

    if(!preg_match("/^([-a-z0-9ÀÁÂÃÄÅÇÈÉÊËÌÍÎÏÒÓÔÕÖÙÚÛÜÝàáâãäåçèéêëìíîïðòóôõöùúûüýÿ_-])+$/i", $input[$field]) !== FALSE)
    {
        return array(
            'field' => $field,
            'value' => $input[$field],
            'rule'  => __FUNCTION__
        );
    }
}
```

</code>

In short, I have just added in the regular expressions the following string :  "ÀÁÂÃÄÅÇÈÉÊËÌÍÎÏÒÓÔÕÖÙÚÛÜÝàáâãäåçèéêëìíîïñðòóôõöùúûüýÿ"

The affected methods are : validate_alpha, validate_alpha_numeric and validate_alpha_dash.

Note that this change would be useful for Spanish websites as well as for some other euroopean languages.

Regards,

Ben.

**Comment by weogrim** [comment]
Hi, 

I try add polish chars to validation rules. I modifet rules, for example: 

```
/^([a-ząęćłńóśżźĄĘĆŁŃÓŚŻŹÀÁÂÃÄÅÇÈÉÊËÌÍÎÏÒÓÔÕÖÙÚÛÜÝàáâãäåçèéêëìíîïðòóôõöùúûüýÿ])+$/i
```

i try edit modified to iu, but i alwas have 500 in ajax request

Can you provide regex with polish chars: ą ę ć ł ń ó ś ż ź Ą Ę Ć Ł Ń Ó Ś Ż Ź ?
